replicated_kv_store/client.go

```diff
@@ -11,49 +11,64 @@ import (
 )
 
 //WriteCommand is called when the client sends the replica a write request.
-func (node *RaftNode) WriteCommand(operation []string) bool {
+func (node *RaftNode) WriteCommand(operation []string, client string) bool {
 
 	// Perform operation only if leader
-	if node.state == Leader {
+	i := 0
+	oper, val := node.trackMessage[client]
+	if val { //check if entry exists; if it does check if its the same as the one that was previously
+		for i < 3 {
```

Comment:
You can try using reflect.DeepEqual here instead of comparing elements one by one